crates/transaction-generator-lib/src/publishing/publish_util.rs

```diff
@@ -125,6 +129,23 @@ impl Package {
         Self::Simple(modules, metadata)
     }
 
+    pub fn script(publisher: AccountAddress) -> TransactionPayload {
+        let code = &*raw_module_data::SCRIPT_SIMPLE;
+        let mut script = CompiledScript::deserialize(code).expect("Script must deserialize");
+
+        // Change the constant to the sender's address to change script's hash.
```

Comment:
Do you want transactions from the same sender to be the same, or do you want every transaction to be different? If later,  you can use random address here instead?